var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -82,7 +82,7 @@ class Geant4(CMakePackage):
         depends_on('xerces-c netaccessor=curl cxxstd=' + std, when='cxxstd=' + std)
 
         # Vecgeom specific versions for each Geant4 version
-        depends_on('vecgeom@1.1.8 cxxstd=' + std,
+        depends_on('vecgeom@1.1.8: cxxstd=' + std,
```

Comment:
```suggestion
        depends_on('vecgeom@1.1.8:1.1 cxxstd=' + std,
```

This is valid syntax, 1.1.8: also includes 1.2.0, 2.0.0, ..., infinity

Reply:
I should get off of GitHub today... That's exactly what I wanted to include in the PR, but somehow I hadn't committed the final change from `1.1.8:` to `1.1.8:1.1` ...